### ISSUE TITLE
Option to define the ticks range, max number of ticks and distribution on Y -axis on carbon SimpleBarChart

### ISSUE THREAD

**Issue by reeteshsingh93** [carbon-design-system/carbon-charts#483]
#### I am submitting a...
 Feature request:

Do we have any option to define the ticks range, max number of ticks and distribution on Y -axis on carbon SimpleBarChart?

#### charts version:

    "@carbon/charts": "^0.24.3",
    "@carbon/charts-react": "^0.24.3"

### Issue description

Unable to set ticks distribution which allows the charts to auto decide and hence it looks squashed.
![image](https://user-images.githubusercontent.com/46310374/73946158-7e4ad700-491b-11ea-8fce-4b3caabafd96.png)

#### Expected behavior

Option to define the distribution on Y -axis on carbon SimpleBarChart as we have in d3.

**Comment by theiliad** [comment]
This is part of what Accurat is working on, going to assign them

**Comment by SerenaG** [comment]
Hi @theiliad, 
Accurat here!

This is a repro case of this issue: https://codesandbox.io/s/carbon-charts-repro-issue-483-4fbiq

The option to define the number of ticks on the axis is already implemented: 
https://github.com/carbon-design-system/carbon-charts/blob/master/packages/core/src/interfaces/axis-scales.ts#L38
Can be used to manually fix this problem: https://codesandbox.io/s/carbon-charts-repro-issue-483-workaround-n018u

An improvement could be to enhance the Y axis to show a correct number of ticks depending on the chart height, do you think it would be useful?

**Comment by caesarsol** [comment]
Status: 
The vertical ticks enhancement is working.
We are now applying the same logic on the horizontal ticks, as requested by @theiliad.

**Comment by theiliad** [comment]
Let me know how you'd like to move forward. My position is that ideally we'd want an axis to figure out how many labels to render regardless of position.
